drf_kit/cache.py

```diff
@@ -100,5 +119,21 @@ def process_cache_response(
 
         return response
 
+    def _get_cached_result(self, key, request) -> dict | None:
+        try:
+            return self.cache.get(key)
+        except Exception as exc:
```

Comment:
We should handle specific errors here, and for this we must test extensively what can go wrong.
Exception is too broad and it might suppress some dumb coding errors such as AttributeError, which is easily fixed and not related to cache itself.

Reply:
I get that handling `Exception` is not the best practice when the code block inside `try` is proprietary code, but actually `self.cache.get` calls a cache inside user code (as cache is defined by them). Therefore, by passing the exception to `cache_error_func`, we are giving the user the power to deal with whatever cache-specific error it might have.

However, as you said, I do see that user might have configured cache just wrong, and both `self.cache`, `self.cache.get` can be None (leading to `AttributeError`) or `self.cache.get` might require a different number of parameters (leading to a `TypeError`). In these two cases, I do see benefit in handling exceptions inside drf-toolkit code. Else, I think we should let the user handle whatever error occurs (thanks for the suggestion!).

Besides, considering we might not want to hide errors from someone who should know about it, we can make this "ignore error" behavior a default disabled? That way, someone has consciously choose to ignore errors (or handle them in a custom manner). Wdyt?